include/rnp/rnp.h

```diff
@@ -2212,7 +2212,8 @@ RNP_API rnp_result_t rnp_op_sign_set_hash(rnp_op_sign_t op, const char *hash);
 /** @brief Set signature creation time. By default current time is used.
  *  @param op opaque signing context. Must be successfully initialized with one of the
  *         rnp_op_sign_*_create functions.
- *  @param create creation time in seconds since Jan, 1 1970 UTC
+ *  @param create creation time in seconds since Jan, 1 1970 UTC. It is 32-bit uint since in
+ *                OpenPGP messages it is stored as 32-bit uint as well.
```

Comment:
I think this description needs some better wording like "32 bit unsigned integer datatype is used here because timestamps in OpenPGP messages are stored as 32-bit unsigned integers as well" or something like that.

Also what does it try to explain, why 64-bit integer is not used here instead?